tests/VMelnalksnis.PaperlessDotNet.Tests.Integration/Documents/DocumentClientTests.cs

```diff
@@ -94,6 +95,21 @@ public async Task Create()
 
 		updatedDocument.Title.Should().Be(update.Title);
 
+		var documentDownload = await Client.Documents.Download(id, original: true);
```

Comment:
If you are adding the other download methods as well, might we worth moving this to a separate test method.

Reply:
I created a separate method to test all of the download methods. I was not sure if I should create a separate method for each operation (download/preview/thumbnail) and thought one test case would be enough. 

What would you prefer?